app/views/trainees/show.html.erb

```diff
@@ -62,7 +62,7 @@
       component.slot(
         :row,
         task_name: 'Degree',
-        path: trainee_degrees_new_type_path(@trainee),
+        path: @trainee.degrees.present?  ? trainee_degrees_confirm_path(@trainee) : trainee_degrees_new_type_path(@trainee),
```

Comment:
`degrees` is already available via `@degrees` so no need to query through a `trainee`

Reply:
![image](https://user-images.githubusercontent.com/470137/97583598-08c5f000-19ef-11eb-9de2-125396e287ac.png)